src/compiler/compile/nodes/Element.ts

```diff
@@ -496,6 +506,18 @@ export default class Element extends Node {
 				}
 			}
 		}
+
+		if (a11y_no_onchange.has(this.name)) {
+			const change_handler = handlers_map.get('change');
+			const blur_handler = handlers_map.get('blur');
+
+			if (change_handler && !blur_handler) {
+				component.warn(this, {
+					code: `a11y-no-onchange`,
+					message: `A11y: onBlur must be used instead of onchange, unless absolutely necessary and it causes no negative consequences for keyboard only or screen reader users.`
```

Comment:
This message should probably be updated to reflect actual Svelte syntax - `on:blur`, `on:change`

Reply:
ok, solved